.changeset/great-birds-wait.md

```diff
@@ -0,0 +1,7 @@
+---
+'@sap-ux-private/control-property-editor-common': minor
+'@sap-ux-private/preview-middleware-client': minor
+'@sap-ux/control-property-editor': minor
```

Comment:
Changes to this packages require also a changeset for the `preview-middleware`. Please add changeset.

Reply:
ok Zdravko.

Reply:
done

Reply:
thanks